src/triage/component/catwalk/storage.py

```diff
@@ -27,9 +30,15 @@ def exists(self):
         raise NotImplementedError
 
     def load(self):
-        raise NotImplementedError
+        with self.open('rb') as f:
+            return f.read()
 
-    def write(self, obj):
+    def write(self, bytestream):
+        with self.open('wb') as f:
+            f.write(bytestream)
+
+    @contextmanager
```

Comment:
Though this is an abstract base method, I'm not sure there's anything gained by defining it as a context manager either.